.circleci/config.yml

```diff
@@ -40,9 +40,15 @@ workflows:
       - e2e_classroom_page:
           requires:
             - e2e_file_upload_features
+      - e2e_topic_and_story_editor_file_upload_features:
+          requires:
+            - e2e_file_upload_features
```

Comment:
Just a heads up that #9523 is going to change this.

Reply:
Ok if that gets merged first, i'll fix the merge conflicts.